src/HotChocolate/Fusion-vnext/src/Fusion.Composition/SatisfiabilityValidator.cs

```diff
@@ -255,6 +325,42 @@ private ImmutableArray<SatisfiabilityError> ValidateSourceSchemaTransition(
 
         return [.. errors];
     }
+
+    private List<IDirective> GetLookupsById(MutableObjectTypeDefinition type)
```

Comment:
Shall we make this an extension method, like `GetFusionLookupDirectives` is? (I can do this if you like)